vertical-pod-autoscaler/pkg/recommender/routines/recommender.go

```diff
@@ -102,22 +102,27 @@ func (r *recommender) UpdateVPAs() {
 		vpa.Recommendation = getCappedRecommendation(vpa.ID, resources, observedVpa.Spec.ResourcePolicy)
 		// Set RecommendationProvided if recommendation not empty.
 		if len(vpa.Recommendation.ContainerRecommendations) > 0 {
-			vpa.Conditions.Set(vpa_types.RecommendationProvided, true, "", "")
 			if !had {
```

Comment:
Could you please merge these two lines into a single if?

Reply:
Done